app/views/catalog/_reviewer_action.html.erb

```diff
@@ -0,0 +1,3 @@
+<% unless document.reviewed %>
+  <%= link_to "Mark as Reviewed", review_reviewer_path(:id => document.id), :method => :patch, class: "reviewer-link" %>
```

Comment:
We like hash rockets (`:class => "reviewer-link"`), but even more importantly, we don't want to mix hash rockets with the 1.9 hash syntax.